oda_api/api.py

```diff
@@ -154,6 +157,11 @@ def func_wrapper(*args, **kwargs):
 
 
 class DispatcherAPI:
+
+    # allowing token discovery by default changes the user interface in some cases, 
+    # but in desirable way
+    token_discovery_methods = None
+
```

Comment:
The changes you talk about are related to the fact that the token might issued even for cases where it should not?

Reply:
not at all. This would not be desirable.

What is implemented is, in fact, desirable.

Also, this part is not related to generating token. 
Only to discovering token.
The change is that now, if there is a discoverable (in env variable, or a file) token, it will be used by default. Before, token was only used if explicitly provided.

So now it is possible to preserve user own code with no explicit reference to tokens, and provide token at runtime. Less work for user, and less chance to embed token in shared code.

Reply:
Ok, that's indeed what I was thinking, and it makes sense.